src/clj/web/lobby.clj

```diff
@@ -219,7 +238,7 @@
    (let [lobbies (app-state/get-lobbies)]
      (doseq [[uid ev] (prepare-lobby-list lobbies users)]
        (when uid
-         (ws/chsk-send! uid ev))))))
+         (future (ws/chsk-send! uid ev)))))))
```

Comment:
this could blow up if there's many messages. maybe we should use pmap or a custom executor.

Reply:
yeah, makes sense. I'll figure it out tonight.